### PR TITLE
Screen template

### PR DESCRIPTION
**Pull request by mwarger**
updated to use screen-style instead of saga
fixed typo in tests

### CODE REVIEW

**Review by skellock — CHANGES_REQUESTED** [review]
Thanks.  Could you run the linter on this please?

**Comment by mwarger** [comment]
Linting completed.

**Comment by mwarger** [comment]
@skellock Please let me know if there's anything else I should do.  Thanks!

File: commands/screen.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 // @cliDescription  Generates an opinionated container.
 
-module.exports = async function (context) {
+module.exports = async function(context) {
```
Comment (skellock) [inline commands/screen.js]: This needs a space between function and (context).  You can run `npm run standard` to find these issues.

File: commands/screen.js [hunk 2]
```diff
@@ -10,7 +10,7 @@ module.exports = async function (context) {
     print.info(`${context.runtime.brand} generate saga <name>\n`)
     print.info('A name is required.')
     return
-  }
+  }
```
Comment (skellock) [inline commands/screen.js]: This trailing space will be picked up by the linter.

**Review by skellock — CHANGES_REQUESTED** [review]

**Comment by skellock** [comment]
Looks like that file was missed by the linting.  If you could patch that up, we're good to go!

Thank you kindly.

**Comment by mwarger** [comment]
Ok, it looks like it passed linting.  Thanks for your help!

**Review by skellock — APPROVED** [review]

**Comment by skellock** [comment]
Thank you sir!

**Pull request merged**